lib/adapters/signature-help-adapter.ts

```diff
@@ -48,7 +47,7 @@ export default class SignatureHelpAdapter {
   }
 
   // Public: Retrieves signature help for a given editor and position.
-  getSignatureHelp(editor: atom$TextEditor, point: atom$Point): Promise<?atomIde$SignatureHelp> {
+  public getSignatureHelp(editor: TextEditor, point: Point): SignatureHelp | Promise<SignatureHelp> {
```

Comment:
Shouldn't it be `Promise<lsp.SignatureHelp | null>`?

Reply:
Good catch, thanks!